themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1792,6 +1791,12 @@ html body.tc-body.tc-single-tiddler-window {
 	width: 100%;
 }
 
+.tc-controlpanel svg.tc-image-button{
+	width: 1em;
+	height: 1em;
```

Comment:
set your browser zoom to 100% and imo it will be too small. so I'd suggest. 1.5em